test/tee_pipeline_test.exs

```diff
@@ -15,24 +15,25 @@ defmodule TestPipeline do
     Pipeline.start_link(%Pipeline.Options{
```

Comment:
This file should be called `test/membrane_element_tee/master_test.exs` and the module `Membrane.Element.Tee.MasterTest`